crates/pathfinder/src/state/sync.rs

```diff
@@ -31,12 +31,15 @@ use tokio::sync::{mpsc, RwLock};
 
 pub struct State {
     pub status: RwLock<SyncStatus>,
+    /// Latest known StarkNet version.
+    pub version: RwLock<Option<String>>,
 }
```

Comment:
Would consider evolving this towards `RwLock<State>` but right now ... It doesn't really matter.

Reply:
I opted for something less intrusive on the rest of the code; potentially this will get deleted once nonces are supported correctly.